### PR TITLE
[CRT] Enable Orchestrator

### PR DESCRIPTION
**Pull request by curtbushko**
Changes proposed in this PR:

- This PR needs to be merged before I can start testing the orchestrator
- Note: This will not break anything as it is the orchestrator that needs these files
- Turning on builds, slack notifications, signing/notarizing, security scanning of artifacts we create 

How I've tested this PR:

- Will test post merge and probably make changes

How I expect reviewers to test this PR:

👀 

Checklist:
- [ ] Tests added
- [ ] CHANGELOG entry added 
  > HashiCorp engineers only, community PRs should not add a changelog entry.
  > Entries should use present tense (e.g. Add support for...)

### CODE REVIEW

File: .release/ci.hcl [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+  // i.e. send "merge" complete signal to orchestrator to trigger build
+}
+
+event "build" {
+  depends = ["merge"]
+  action "build" {
+    organization = "hashicorp"
+    repository = "consul-k8s"
+    workflow = "build"
+  }
```
Comment (thisisnotashwin) [inline .release/ci.hcl:30]: should we trigger a notification if the build fails?

File: .release/ci.hcl [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+    workflow = "upload-dev"
+    depends = ["build"]
+  }
+
+  notification {
+    on = "fail"
+  }
+}
+
+event "promote-dev-docker" {
```
Comment (thisisnotashwin) [inline .release/ci.hcl:47]: for readability, can we move this event definition to be after the `verify` definition

File: .release/ci.hcl [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+  slack {
+    notification_channel = "CBXF3CGAF" # team-consul-kubernetes
+  }
+  github {
+    organization = "hashicorp"
+    repository = "consul-k8s"
+    release_branches = [
+      # The CRT tool does not support * yes as a branch name
+      "main",
+      "cb/crt-testing"
```
[the remark this answers is not in the document]
Comment (curtbushko) [reply .release/ci.hcl:14]: I am keeping that for the next stage of testing. I will remove it after testing in the next PR

File: .release/ci.hcl (hunk 1 above)
Comment (curtbushko) [reply .release/ci.hcl:30]: If the build fails it should come through the regular slack messages we receive when building.

File: .release/ci.hcl [hunk 4]
```diff
@@ -0,0 +1,271 @@
...
+project "consul-k8s" {
+  team = "consul-k8s"
+  slack {
+    notification_channel = "CBXF3CGAF" # team-consul-kubernetes
+  }
+  github {
+    organization = "hashicorp"
+    repository = "consul-k8s"
+    release_branches = [
+      # The CRT tool does not support * yes as a branch name
```
Comment (t-eckert) [inline .release/ci.hcl:12]:
```suggestion
      # The CRT tool does not support * yet as a branch name
```

Is this a typo?

File: .release/ci.hcl [hunk 5]
```diff
@@ -0,0 +1,271 @@
...
+  action "notarize-darwin-arm64" {
+    organization = "hashicorp"
+    repository = "crt-workflows-common"
+    workflow = "notarize-darwin-arm64"
+  }
+
+  notification {
+    on = "fail"
+  }
+}
```
Comment (t-eckert) [inline .release/ci.hcl:99]: Is this a duplicate of the above? It looks like it depends on itself too.

Comment (t-eckert) [reply .release/ci.hcl:99]: Oh no. `amd` vs `arm`

**Pull request merged**